<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Four Fours</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainLayout">
    <property name="spacing">
     <number>24</number>
    </property>
    <property name="contentsMargins">
     <left>48</left>
     <top>48</top>
     <right>48</right>
     <bottom>48</bottom>
    </property>
    <item>
     <widget class="QLabel" name="LevelTitle">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Inter</family>
        <pointsize>40</pointsize>
        <weight>63</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>LEVEL 1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="EquationField">
      <property name="minimumHeight">
       <number>72</number>
      </property>
      <property name="font">
       <font>
        <family>JetBrains Mono</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>4_4_4_4 = 1</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="padLayout">
      <property name="horizontalSpacing">
       <number>12</number>
      </property>
      <property name="verticalSpacing">
       <number>12</number>
      </property>
      <item row="0" column="0">
       <widget class="QPushButton" name="btnPlus">
        <property name="minimumSize">
         <size>
          <width>84</width>
          <height>72</height>
         </size>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btnMinus">
        <property name="minimumSize">
         <size>
          <width>84</width>
          <height>72</height>
         </size>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btnMul">
        <property name="minimumSize">
         <size>
          <width>84</width>
          <height>72</height>
         </size>
        </property>
        <property name="text">
         <string>*</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="btnDiv">
        <property name="minimumSize">
         <size>
          <width>84</width>
          <height>72</height>
         </size>
        </property>
        <property name="text">
         <string>/</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="btnSqrt">
        <property name="minimumSize">
         <size>
          <width>84</width>
          <height>72</height>
         </size>
        </property>
        <property name="text">
         <string>√</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="btnFAC">
        <property name="minimumSize">
         <size>
          <width>84</width>
          <height>72</height>
         </size>
        </property>
        <property name="text">
         <string>!</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="btnDUBFAC">
        <property name="minimumSize">
         <size>
          <width>84</width>
          <height>72</height>
         </size>
        </property>
        <property name="text">
         <string>!!</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="btnSUBFAC">
        <property name="minimumSize">
         <size>
          <width>84</width>
          <height>72</height>
         </size>
        </property>
        <property name="text">
         <string>!!!</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="btnSCOB1">
        <property name="minimumSize">
         <size>
          <width>84</width>
          <height>72</height>
         </size>
        </property>
        <property name="text">
         <string>(</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="btnSCOB2">
        <property name="minimumSize">
         <size>
          <width>84</width>
          <height>72</height>
         </size>
        </property>
        <property name="text">
         <string>)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="2">
       <widget class="QPushButton" name="btnBack">
        <property name="minimumSize">
         <size>
          <width>84</width>
          <height>72</height>
         </size>
        </property>
        <property name="text">
         <string>⌫</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="actionsLayout">
      <property name="spacing">
       <number>16</number>
      </property>
      <item>
       <widget class="QPushButton" name="Restart">
        <property name="minimumHeight">
         <number>56</number>
        </property>
        <property name="text">
         <string>Перезапуск</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="RestartThis">
        <property name="minimumHeight">
         <number>56</number>
        </property>
        <property name="text">
         <string>Перезапустить уровень</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="CheckBtn">
        <property name="minimumSize">
         <size>
          <width>220</width>
          <height>64</height>
         </size>
        </property>
        <property name="text">
         <string>ПРОВЕРИТЬ</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
